caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -151,18 +210,13 @@ export default function decorate(block) {
   }
 
   function setPositionByIndex() {
```

Comment:
Move this function ouside of default function and pass in what you need instead of relying variables that have been defined outside of the function. This ensures the function is given what it needs and will reduce unintended bugs.